logic/peers.js

```diff
@@ -43,20 +43,28 @@ var modules;
  * @todo Add description for the params
  */
 // Constructor
-function Peers(logger, cb) {
-	library = {
-		logger,
-	};
-	self = this;
-	__private.me = null;
+class Peers {
+	constructor(logger, cb) {
+		library = {
+			logger,
+		};
+		self = this;
+		__private.me = null;
 
-	this.peersManager = peersManager;
+		this.peersManager = peersManager;
 
-	return setImmediate(cb, null, this);
+		return setImmediate(cb, null, this);
+	}
 }
 
+// TODO: TO maintain backward compatibility, have to user prototype otherwise these must be converted to static attributes
+/**
+ * Returns current peer state and system headers.
+ *
+ * @returns {peer} Clone of peer
+ */
 Peers.prototype.me = function() {
-	var me = _.extend(System.getHeaders(), { state: Peer.STATE.CONNECTED });
+	const me = _.extend(System.getHeaders(), { state: Peer.STATE.CONNECTED });
```

Comment:
Couldn't we use the object spread operator here instead of lodash extend??

Reply:
Not in Node v6.

Reply:
@willclarktech

Reply:
You're right!

Reply:
Could use `Object.assign({}, System.getHeaders(), { state: Peer.STATE.CONNECTED });` though.